mappings/net/minecraft/entity/FallingBlockEntity.mapping

```diff
@@ -20,6 +20,11 @@ CLASS net/minecraft/class_1540 net/minecraft/entity/FallingBlockEntity
 		ARG 2 pos
 	METHOD method_32879 (Lnet/minecraft/class_1282;FLnet/minecraft/class_1297;)V
 		ARG 2 entity
+	METHOD method_40005 spawnFromBlock (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;)Lnet/minecraft/class_1540;
+		COMMENT Spawns a falling block entity at {@code pos} from the block {@code state}.
```

Comment:
This should document that the spawned entity is returned.

Reply:
@haykam821 Can I just write "and returns..."? Do I use `@returns`?

Reply:
`@return the spawned entity` seems fine to me